ejdb/utils.py

```diff
@@ -112,7 +112,7 @@ def read_ejdb_config():
     parser = ConfigParser()
     parser.read([os.path.expanduser('~/.ejdb.cfg'), config_path])
     try:
-        return parser['ejdb']['path']
+        return parser.get('ejdb', 'path')
     except KeyError:
```

Comment:
This exception needs to be fixed as well. `ConfigParser.get` raises `NoSectionError` and `NoOptionError`, not `KeyError`.